src/message/html/js.js

```diff
@@ -1,21 +1,39 @@
 /* eslint-disable */
 
 export default `
-window.onerror = function(message){
-   alert('error');
-   return false;
- };
+window.onerror = function(message) {
+  alert(message);
+  return false;
+};
 
 document.addEventListener('message', function(e) {
   const msg = JSON.parse(e.data);
-  switch (msgObj.type) {
+  switch (msg.type) {
+    case 'loading-top':
+      var element = document.getElementById('top_loader');
+      if (msg.newState) {
+        element.classList.add('loading-spinner');
+        window.scrollTo(0, 0);
+      } else {
+        element.classList.remove('loading-spinner');
+      }
+      break;
+    case 'loading-bottom':
+      var element = document.getElementById('bottom_loader');
+      if (msg.newState) {
+        element.classList.add('loading-spinner');
+        window.scrollTo(0, document.body.scrollHeight);
```

Comment:
I don't think you should be scrolling here either.
We can have a separate scrollToBottom event that will rarely if ever combine with 'loading-bottom'.